src/EnergyPlus/CurveManager.cc

```diff
@@ -212,16 +207,15 @@ namespace CurveManager {
             ShowFatalError(state, "CurveValue: Invalid Interpolation Type");
```

Comment:
This should probably be an `assert(false);`.

Reply:
Yeah...I think as I continue the refactor and add inheritance, this will simply vanish.